actors/multisig/tests/util.rs

```diff
@@ -43,4 +50,53 @@ impl ActorHarness {
         rt.verify();
         ret
     }
+
+    pub fn propose_ok(
+        self: &Self,
+        rt: &mut MockRuntime,
+        to: Address,
+        value: TokenAmount,
+        method: MethodNum,
+        params: RawBytes,
+    ) -> [u8; 32] {
+        //expect_ok(self.propose(rt, to, value, method, params));
+        rt.expect_validate_caller_type(vec![*ACCOUNT_ACTOR_CODE_ID, *MULTISIG_ACTOR_CODE_ID]);
+        let propose_params =
+            ProposeParams { to: to, value: value.clone(), method: method, params: params.clone() };
+        expect_ok(
+            rt.call::<Actor>(Method::Propose as u64, &RawBytes::serialize(propose_params).unwrap()),
+        );
+        // compute proposal hash
+        let txn = Transaction {
+            to: to,
+            value: value,
+            method: method,
+            params: params,
+            approved: vec![rt.caller],
+        };
+        compute_proposal_hash(&txn, rt).unwrap()
+    }
+
+    pub fn assert_transactions(
+        self: &Self,
+        rt: &MockRuntime,
+        expect_txns: HashMap<TxnID, Transaction>,
+    ) {
+        let st = rt.get_state::<State>().unwrap();
+        let ptx = make_map_with_root::<_, Transaction>(&st.pending_txs, &rt.store).unwrap();
+        let expect_count = expect_txns.len();
+        // check that all expected txns exist in state
+        for (txn_id, expect_v) in expect_txns {
+            let v = ptx.get(&txn_id.key()).unwrap().unwrap();
+            assert_eq!(expect_v, *v);
+        }
+        // check that there are no more txns in state than in expected
+        let mut count = 0;
+        ptx.for_each(|_tx_id, _txn: &Transaction| {
+            count += 1;
+            Ok(())
+        })
+        .unwrap();
```

Comment:
I don't know if this unwrapping is idiomatic.  I've been using it as a way to assert no error.  Maybe matching and panicking would be better? This is easy but it looks kind of ugly and I seem to be breaking the pattern of everything returning a `Result`.

Reply:
I think it's fine for test methods where there's otherwise nothing to return, I've been doing similar.